package.json

```diff
@@ -10,8 +10,9 @@
         "frontend:reinstall": "cd frontend && npm reinstall",
         "backend:reinstall": "cd backend && npm reinstall",
         "start": "concurrently npm:frontend npm:backend -n frontend,backend -c green,blue",
-        "plugins": "concurrently npm:backend npm:serve:plugins -c green,blue,magenta",
+        "plugins": "concurrently npm:backend npm:serve:plugins npm:ocp-console -n frontend,backend,console -c green,blue,magenta",
```

Comment:
frontend and backend are in the wrong order here

Reply:
fixed